test/CMakeLists.txt

```diff
@@ -44,6 +44,7 @@ wdk_add_driver(kf-test WINVER NTDDI_WIN10 STL
     HexTest.cpp
     MapTest.cpp
     Vector.cpp
+    IntrusivePtr.cpp
```

Comment:
We forgot to add Test suffix to the files.Let's add it.

Reply:
I thought about opposite - you've added Vector.cpp, so I thought that the Test prefix is extra. The same was done by @Kellesi here: https://github.com/apriorit/kf/pull/35/files